server/sonar-web/src/main/js/apps/overview/stickers/utils.ts

```diff
@@ -0,0 +1,63 @@
+///
+/// SonarQube
+/// Copyright (C) 2009-2018 SonarSource SA
+/// mailto:info AT sonarsource DOT com
+///
+/// This program is free software; you can redistribute it and/or
+/// modify it under the terms of the GNU Lesser General Public
+/// License as published by the Free Software Foundation; either
+/// version 3 of the License, or (at your option) any later version.
+///
+/// This program is distributed in the hope that it will be useful,
+/// but WITHOUT ANY WARRANTY; without even the implied warranty of
+/// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+/// Lesser General Public License for more details.
+///
+/// You should have received a copy of the GNU Lesser General Public License
+/// along with this program; if not, write to the Free Software Foundation,
+/// Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+///
+
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import { getHostUrl } from '../../../helpers/urls';
+
+export type StickerColors = 'white' | 'black' | 'orange';
+
+export interface StickerOptions {
+  color: StickerColors;
+}
+
+export enum StickerType {
+  badge = 'badge',
+  card = 'card',
+  marketing = 'marketing'
+}
+
+export function getStickerUrl(type: StickerType, options: StickerOptions) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this function to use "return" consistently ([typescript:S3801](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3801))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fgreg%2Fscanned_on_sonarcloud_badge&issues=AWDmQjl7GaMZFZs6hjD6&open=AWDmQjl7GaMZFZs6hjD6)

[ISSUE_KEY]: AWDmQjl7GaMZFZs6hjD6